.github/workflows/cmd_template.yml

```diff
@@ -33,6 +33,11 @@ jobs:
           pip install -r .github/workflows/requirements.txt
           pip install -r .github/workflows/requirements${{ matrix.python-version }}.txt
 
+      - name: Install smoderp2d
+        run: |
+          pip wheel .
+          pip install smoderp2d-*.whl
```

Comment:
Why not calling 

```
pip install .
```

instead of

```
pip wheel .
pip install smoderp2d-*.whl
```
?

Reply:
It used to cause some problems at my side but the problems were apparently solved in some other commits as I don't face them anymore. Changed to yours in [761fc34](https://github.com/storm-fsv-cvut/smoderp2d/pull/325/commits/761fc34ce88a682ee9a5fe72e6f90416db7cbe2b).